test/tests/mesh/checkpoint/tests

```diff
@@ -6,6 +6,7 @@
     cli_args = 'Outputs/file_base=test_2  Mesh/file=checkpoint_split_in.2.cpr'
     max_parallel = 2  # workaround until libMesh/libmesh#1408
     min_parallel = 2  # workaround until libMesh/libmesh#1409
+    skip = 'Requires libMesh/libmesh#1409'
```

Comment:
Do we want to skip them completely or just skip them on Mac? They work fine on the linux boxes.